MdsSupportingClasses/Collivery.php

```diff
@@ -114,6 +114,8 @@ private function consumeAPI($url, $data, $type, $isAuthenticating = false) {
             $client = curl_init($url.'?'.$query);
         }
 
+        curl_setopt($client, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
Stop disabling this!
There's no reason to disable it even in your local environment